.github/workflows/qa.yml

```diff
@@ -0,0 +1,158 @@
+name: Quality Assurance
+
+on:
+  push:
+    branches: [master, dev]
+  pull_request:
+    branches: [master, dev]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+        with:
+          submodules: 'recursive'
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+
+      - name: Use Node.js (.nvmrc)
+        uses: actions/setup-node@master
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+
+      - name: Cache node modules
```

Comment:
Funguje ta cache? Ja mam pocit ze instalacia trva 30+ sekund bez ohladu na cache, takze to tu mame zbytocne. [Vid moja otazka tu.](https://github.com/priznanie-digital/priznanie-digital/pull/169#discussion_r448172630)

A cache pridava zbytocnych dalsich par sekund, niekedy az 13!
![Screenshot 2020-07-06 at 11 04 20](https://user-images.githubusercontent.com/1561771/86576269-cf1d3500-bf78-11ea-9d39-c5e98b02241e.png)

Reply:
Navrhujem toto poskusat v dalsom PR

Reply:
Suhlasim a preto som chcel aby sa cache nepridavala teraz ale skusala v dalsom PR 🙃